metadata.json

```diff
@@ -19,6 +19,14 @@
         "10"
       ]
     },
+    {
+      "operatingsystem": "FreeBSD",
+      "operatingsystemrelease": [
+        "11",
```

Comment:
FreeBSD 11 is expected EOL in 2 months - https://www.freebsd.org/security/#sup

Reply:
Good catch, let me remove it :-)